content/backmatter.tex

```diff
@@ -437,7 +437,9 @@ \subsection{\Fortran: SHMEM\_PUT}
 \FUNC{SHMEM\_PUT} is ambiguous and has been deprecated.
 
 \subsection{SHMEM\_CACHE}
-The \FUNC{SHMEM\_CACHE} \ac{API}
+The \FUNC{SHMEM\_CACHE} \ac{API}, listed below, was originally implemented for systems with cache management instructions.
+This API has largely gone unused on cache-coherent system architectures.
+\FUNC{SHMEM\_CACHE} has been deprecated.
```

Comment:
I personally do not like this stylistic change because I prefer the listed items to be inlined with the description instead of separate.

Reply:
I'd vote for leaving in the original location as well.